### PR TITLE
ci: add new advisory to audit-ci

### PR DESCRIPTION
**Pull request by fionnachan**

### CODE REVIEW

File: audit-ci.jsonc [hunk 1]
```diff
@@ -54,6 +54,10 @@
     // https://github.com/advisories/GHSA-4h98-2769-gh6h
     // OpenZeppelin Contracts vulnerable to ECDSA signature malleability
     // We don’t use signatures for replay protection anywhere
-    "GHSA-4h98-2769-gh6h"
+    "GHSA-4h98-2769-gh6h",
+    // https://github.com/advisories/GHSA-xvch-5gv4-984h
+    // Minimist <=1.2.5 is vulnerable to Prototype Pollution via file index.js, function setKey() (lines 69-95).
+    // GHSA-xvch-5gv4-984h|typedoc-plugin-markdown>handlebars>minimist
```
Comment (yahgwai) [inline audit-ci.jsonc:60]: When we add an advisory we also add a justification as to why it's ok to ignore

**Comment by yahgwai** [comment]
Lets also add @fredlacs to this PR, it's good to get multiple eyes when adding a new advisory bypass

File: audit-ci.jsonc (hunk 1 above)
Comment (fionnachan) [reply audit-ci.jsonc:60]: I'd put 
> Minimist is a typedoc plugin's dependency and is never included in the sdk build. Typedoc was only called by the nitro-docs repo to generate sdk docs.

any comments?

Comment (yahgwai) [reply audit-ci.jsonc:60]: Yeh, I think that's good. Maybe mention the words "dev dependency".

**Review by yahgwai — APPROVED** [review]

**Pull request merged**